### DISCUSSION TITLE
whisper support 185H Ultra Core cpu for acceleration?

### DISCUSSION THREAD

**Discussion by dittoda** [openai/whisper#2224] [Q&A]
Even though I tried to find out about this, I could check only something about Utra Core 7 (155H) and nothing talking about whisper (maybe this is a  natural thing, because whisper is just openai's product.) 

https://download.intel.com/newsroom/2023/ai/ai-everywhere-2023/Intel-Core-Ultra-Processors-Media-Presentation.pdf

Anyway may I know I can go with 185H for using whisper's acceleration?

**Comment by phineas-pta** [comment]
whisper works on most cpu

but dont expect anything outstanding when running on cpu, u need gpu for anything AI
